common/djangoapps/enrollment/urls.py

```diff
@@ -3,7 +3,8 @@
 
 """
 from django.conf import settings
-from django.conf.urls import patterns, include, url
+from django.conf.urls import include
```

Comment:
We're less likely to get conflicts in the future if we break it out onto its own line.